wpimath/src/main/native/cpp/geometry/Rotation3d.cpp

```diff
@@ -235,3 +237,15 @@ units::radian_t Rotation3d::Angle() const {
 Rotation2d Rotation3d::ToRotation2d() const {
   return Rotation2d{Z()};
 }
+
+void frc::to_json(wpi::json& json, const Rotation3d& rotation) {
+  json = wpi::json{{"yaw", rotation.X().value()},
+                   {"pitch", rotation.Y().value()},
+                   {"roll", rotation.Z().value()}};
+}
+
+void frc::from_json(const wpi::json& json, Rotation3d& rotation) {
+  rotation = Rotation3d{units::degree_t{json.at("yaw").get<double>()},
+                        units::degree_t{json.at("pitch").get<double>()},
+                        units::degree_t{json.at("roll").get<double>()}};
```

Comment:
This deserialization is incorrect since the serialized values in to_json() are radians, not degrees. Use units::radian_t here instead.